src/components/Instructions.component.js

```diff
@@ -5,16 +5,25 @@ import './../styles/instructions.css'
 
 class Instructions extends Component {
   render() {
-    return (
-      <div className="instructions">
-        Follow the instructions on the README to get started!
-        <img
-          className="instructions__logo"
-          src="https://uiuc.hack4impact.org/img/colored-logo.png"
-          alt="h4i logo"
-        />
-      </div>
-    )
+    let shouldDisplayImage = this.props.shouldDisplayImage
+    if (shouldDisplayImage) {
+      return (
+        <div className="instructions">
```

Comment:
You've copied this code here and below. instead of having this one massive `if ...` check, and returning the entire tree differently, look at this example that doesn't have code reuse. It's cleaner and shorter and more easily extensible :)

```javascript

render() {
  return (
    <div>
      This text will always show.
      { this.props.shouldDisplayImage && <img src="#" alt="This image only shows up if the prop is true" />}
    <div>
  )